events/schema.py

```diff
@@ -132,6 +140,63 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         return DeleteEventMutation()
 
 
+class EnrolOccurrenceMutation(graphene.relay.ClientIDMutation):
+    class Input:
+        occurrence_id = graphene.GlobalID(
+            required=True, description="Occurrence id " "of event"
+        )
+        child_id = graphene.GlobalID(required=True, description="Guardian's child id")
+
+    enrolment = graphene.Field(EnrolmentNode)
+
+    @classmethod
+    @login_required
+    @transaction.atomic
+    def mutate_and_get_payload(cls, root, info, **kwargs):
+        occurrence_id = from_global_id(kwargs["occurrence_id"])[1]
+        child_id = from_global_id(kwargs["child_id"])[1]
+        user = info.context.user
+        try:
+            occurrence = Occurrence.objects.get(pk=occurrence_id)
+        except Occurrence.DoesNotExist as e:
+            raise KukkuuGraphQLError(e)
+        try:
+            child = Child.objects.user_can_update(user).get(pk=child_id)
+        except Child.DoesNotExist as e:
+            raise KukkuuGraphQLError(e)
+        if child.occurrences.filter(event=occurrence.event).exists():
+            raise KukkuuGraphQLError("Child already joined this event")
+        enrolment = Enrolment.objects.create(child=child, occurrence=occurrence)
+
+        return EnrolOccurrenceMutation(enrolment=enrolment)
+
+
+class UnenrolOccurrenceMutation(graphene.relay.ClientIDMutation):
+    class Input:
+        occurrence_id = graphene.GlobalID(
+            required=True, description="Occurrence id " "of event"
```

Comment:
Ditto.

Reply:
Those must be conflicts between PyCharm auto format and black 💩